app/apiimporter/src/commonMain/kotlin/com/moneymanager/apiimporter/ApiSessionImportService.kt

```
@@ -2195,13 +2196,34 @@ private fun responseItemsArray(
+ */
+internal fun responseCodeOk(
+    json: String,
+    successCodeField: String?,
+    successCodeOkValue: String?,
+): Boolean {
+    if (successCodeField == null) return true
+    return try {
+        val actual =
+            (Json.parseToJsonElement(json).resolveJsonPathElement(successCodeField) as? JsonPrimitive)?.contentOrNull
+        actual == successCodeOkValue
+    } catch (e: SerializationException) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)